test/widget_tests/after_auth_screens/events/create_event_page_test.dart

```diff
@@ -403,31 +403,6 @@ void main() {
           false,
         );
       });
-      testWidgets("Testing Add Admins section", (tester) async {
```

Comment:
@Kushalrock please do not remove an existing test. Done deliberately please ignore with a clarification.

Reply:
That is the test for admins part of the screen which has been removed.